src/components/ControlPanel.vue

```diff
@@ -68,54 +68,11 @@
           </v-flex>
         </v-layout>
 
-        <v-data-table
-          :headers="headers"
-          :items="tableNodes"
-          :footer-props="{
-            itemsPerPageOptions: [10, 20, { text: 'All', value: -1 }]
-          }"
-          :items-per-page.sync="nodeTableItems"
-          item-key="id"
-          class="elevation-1"
-        >
-          <template v-slot:item="{ item }">
-            <tr
-              :style="{
-                cursor: 'pointer',
-                background:
-                  selectedNode === item
-                    ? $vuetify.theme.themes.light.accent
-                    : 'none'
-              }"
-              @click.stop="selectNode(item)"
-            >
-              <td>{{ item.id }}</td>
-              <td>
-                {{ item.ready ? item.manufacturer : '' }}
-              </td>
-              <td>
-                {{ item.ready ? item.productDescription : '' }}
-              </td>
-              <td>
-                {{ item.ready ? item.productLabel : '' }}
-              </td>
-              <td>{{ item.name || '' }}</td>
-              <td>{{ item.loc || '' }}</td>
-              <td>{{ item.isSecure ? 'Yes' : 'No' }}</td>
-              <td>{{ item.isBeaming ? 'Yes' : 'No' }}</td>
-              <td>{{ item.failed ? 'Yes' : 'No' }}</td>
-              <td>{{ item.status }}</td>
-              <td>{{ item.interviewStage }}</td>
-              <td>
-                {{
-                  item.lastActive
-                    ? new Date(item.lastActive).toLocaleString()
-                    : 'Never'
-                }}
-              </td>
-            </tr>
-          </template>
-        </v-data-table>
+        <nodes-table
+          :nodes="nodes"
+          :showHidden="showHidden"
+          v-on:node-selected="selectNode"
```

Comment:
If you press on a node, it is selected on the table but the tab isn't populated with its info

Reply:
Fixed.